lib/features/application/shared_widgets/bottom_navigation_menu/bottom_navigation_menu_mobile_tablet.dart

```diff
@@ -17,11 +17,7 @@ class BottomNavigationMenuMobileTablet extends StatelessWidget {
           child: Column(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: <Widget>[
-              Divider(
```

Comment:
Please don't remove this divider, it is important to separate (in all pages) between the site and the bottom part